app/assets/javascripts/discourse/app/components/modal-container.hbs

```diff
@@ -1,17 +1,33 @@
-<DModal
-  @modalClass={{concat-class
-    this.modal.modalClass
-    (if this.modal.opts.panels "has-tabs")
+<div class="modal-container" {{did-insert this.modal.setContainerElement}}>
```

Comment:
This is where the new component-based modals will actually be rendered (via `{{#in-element}}`)